Documentation/admin_guide.md

```diff
@@ -209,3 +209,11 @@ If the request times out, it indicates two possibilities:
 2. the majority of the cluster is not functioning.
 
 If timeout happens several times continuously, administrators should check status of cluster and resolve it as soon as possible.
+
+### Provisioning
```

Comment:
Cannot decide which to use for this title. In this part, I want to tell people how to set etcd best based on the OS and hardware. One example is that etcd will support two separate disks for log and datastore for best practice, and this will be documented here. Another one is that the memory usage prediction of etcd, so people could assign reasonable RAM to etcd.
Any good naming?
/cc @jonboulle @kelseyhightower @barakmich